server/channels/store/storetest/channel_store.go

```diff
@@ -3908,6 +3922,15 @@ func testChannelStoreGetChannelsByUser(t *testing.T, rctx request.CTX, ss store.
 	require.NoError(t, nErr)
 	require.Len(t, list, 2)
 	require.ElementsMatch(t, []string{o1.Id, o3.Id}, []string{list[0].Id, list[1].Id}, "channels did not match")
+
+	// Archive team and verify channels don't show up
+	team.DeleteAt = model.GetMillis()
+	_, nErr = ss.Team().Update(team)
+	require.NoError(t, nErr)
+
+	// Should return 0 since team is archived
```

Comment:
seems to be that the comment and the expected value for the test are out of sync. And if I understand this correctly, what should be validated should be `require.Len(t, channels, 0)`

Reply:
The comment is wrong yes, but the test is correct since that store method returns an error when there are no rows. I'll fix the comment.